### PR TITLE
Razer Blackwidow Chroma support, Check if Device is BW 2016 or Chroma

### PR DESCRIPTION
**Pull request by Drakenson**
As suggested here my pull request for Blackwidow Chroma support. I've added a check if the device is Chroma or Blackwidow 2016 to enable the backlight feature which seemed to be completely disabled by line 170: 
`if False: # condition for BlackWidow 2016`

If you dislike this addition I can create a new PR with only the added Device ID.

### CODE REVIEW

File: blackwidowcontrol.py [hunk 1]
```diff
@@ -167,7 +173,7 @@ def main():
 		if options.backlight == "unmodified":
 			pass
 		else:
-			if False: # condition for BlackWidow 2016
+			if bw.product_name in BACKLIGHTSUPPORTED_DEVICES: # condition for BlackWidow 2016
```
Comment (Martchus) [inline blackwidowcontrol.py:176]: I think that makes sense - definitely better than the disabled code :-)

File: blackwidowcontrol.py [hunk 2]
```diff
@@ -128,7 +134,7 @@ def main():
 		if options.init == True:
 			LOG("Sending initiation command\n")
 			bw.send(init_old)
-
+
```
Comment (Martchus) [inline blackwidowcontrol.py:137]: Please remove these whitespaces.

**Review by Martchus — COMMENTED** [review]
Thanks, looks almost good to merge. Just remove the whitespaces as mentioned in the inline comment.

**Comment by Drakenson** [comment]
Thanks for your feedback, whitespaces are gone!

**Comment by Martchus** [comment]
Thanks, PR merged.

**Pull request merged**